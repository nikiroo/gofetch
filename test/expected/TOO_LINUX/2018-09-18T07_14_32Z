         VOUS BOSSEZ DANS PHP ? PARTICIPEZ AU BAROMÈTRE 2018       

  2018/09/18

  o News link: https://www.toolinux.com/?Vous-bossez-dans-PHP-Participez-au-barometre-2018
  o Source link: 


    Vous bossez dans PHP ? Participez au baromètre 2018
    
    mardi 18 septembre 2018
    
    Cela  fait  maintenant 5 années que l’AFUP diffuse une campagne
    pour  récolter  des informations sur les salaires et conditions
    de  travail  des  développeuses  et développeurs PHP en France.
    Vous  êtes  invité  à  répondre à l’enquête dès maintenant pour
    l’enquête 2018 : cela prend moins de 5 minutes.
    
    Le  [1]baromètre  des  salaires PHP permet aux développeuses et
    développeurs  d’avoir  «  une  vision  sur la rémunération dans
    leur  secteu  »r.  Il  permet,  entre  autres, de connaître les
    spécialités  les plus attractives, de se comparer par rapport à
    la   moyenne  des  salaires,  ou  de  mettre  en  évidence  des
    tendances régionales.
    
    Avoir  une  enquête  chaque  année  permet de faire un état des
    lieux  régulier  du  secteur,  de  répondre régulièrement à des
    questions  comme le cadre de travail, le type d’entreprise, les
    versions de PHP, et ainsi en suivre l’évolution.
    
    Si  vous avez répondu les années précédentes, vos réponses sont
    tout  aussi  cruciales pour déterminer de telles évolutions, et
    ce   même  si  elles  sont  identiques  à  votre  participation
    précédente.   Plus  il  y  aura  de  réponses,  meilleure  sera
    l’interprétation  des  résultats : nous comptons donc aussi sur
    vous,  qui  n’avez  jamais  répondu  pour  prendre ces quelques
    minutes  afin  de  répondre à l’enquête et ainsi permettre à la
    communauté de mieux se connaître.
    
    [2]Participer à l’enquête
    
    [3]
    
    
    
    [1] https://barometre.afup.org
    
    [2] https://barometre.afup.org/campaign/2018
    
    [3]  https://www.toolinux.com/?Vous-bossez-dans-PHP-Participez-
    au-barometre-2018#forum



