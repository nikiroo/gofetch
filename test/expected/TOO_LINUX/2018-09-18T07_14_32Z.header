0Vous bossez dans PHP ? Participez au baromètre 2018	null/TOO_LINUX/2018-09-18T07_14_32Z		70
i  2018/09/18
i
i    Cela  fait  maintenant 5 années que l’AFUP diffuse une campagne
i    pour  récolter  des informations sur les salaires et conditions
i    de  travail  des  développeuses  et développeurs PHP en France.
i    Vous  êtes  invité  à  répondre à l’enquête dès maintenant pour
i    l’enquête  2018  :  cela prend moins de 5 minutes. Le baromètre
i    des salairesPHP permet aux développeuses et (...)
i
