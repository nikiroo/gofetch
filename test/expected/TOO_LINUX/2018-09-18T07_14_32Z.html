<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>Vous bossez dans PHP ? Participez au baromètre 2018</h1>
	<div class='details'>(2018/09/18)</div>
	<br/>
    <ul>
        <li>News link: <a href='https://www.toolinux.com/?Vous-bossez-dans-PHP-Participez-au-barometre-2018'>https://www.toolinux.com/?Vous-bossez-dans-PHP-Participez-au-barometre-2018</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Vous bossez dans PHP ? Participez au baromètre 2018<br/><br/>mardi 18 septembre 2018<br/><br/>Cela fait maintenant 5 années que l’AFUP diffuse une campagne pour récolter des informations sur les salaires et conditions de travail des développeuses et développeurs PHP en France. Vous êtes invité à répondre à l’enquête dès maintenant pour l’enquête 2018 : cela prend moins de 5 minutes.<br/><br/>Le [1]baromètre des salaires PHP permet aux développeuses et développeurs d’avoir « une vision sur la rémunération dans leur secteu »r. Il permet, entre autres, de connaître les spécialités les plus attractives, de se comparer par rapport à la moyenne des salaires, ou de mettre en évidence des tendances régionales.<br/><br/>Avoir une enquête chaque année permet de faire un état des lieux régulier du secteur, de répondre régulièrement à des questions comme le cadre de travail, le type d’entreprise, les versions de PHP, et ainsi en suivre l’évolution.<br/><br/>Si vous avez répondu les années précédentes, vos réponses sont tout aussi cruciales pour déterminer de telles évolutions, et ce même si elles sont identiques à votre participation précédente. Plus il y aura de réponses, meilleure sera l’interprétation des résultats : nous comptons donc aussi sur vous, qui n’avez jamais répondu pour prendre ces quelques minutes afin de répondre à l’enquête et ainsi permettre à la communauté de mieux se connaître.<br/><br/>[2]Participer à l’enquête<br/><br/>[3]<br/><br/><br/><br/>[1] https://barometre.afup.org<br/><br/>[2] https://barometre.afup.org/campaign/2018<br/><br/>[3] https://www.toolinux.com/?Vous-bossez-dans-PHP-Participez-au-barometre-2018#forum
	</div>
<hr/>
</div>
</body>
